internal/provider/nordvpn/api/getservers.go

```diff
@@ -0,0 +1,80 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/qdm12/gluetun/internal/constants/vpn"
+	"github.com/qdm12/gluetun/internal/models"
+)
+
+var (
+	ErrHTTPStatusCodeNotOK = errors.New("HTTP status code not OK")
+)
+
+func (api *API) GetServers(recommended bool) (servers []models.Server, err error) {
+	var url string
+	if recommended {
+		url = baseURL + "servers/recommendations?limit=10000"
+	} else {
+		url = baseURL + "servers?limit=10000"
+	}
+
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := api.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("%w: %s", ErrHTTPStatusCodeNotOK, res.Status)
+	}
+
+	return parse(res.Body)
+}
+
+func parse(rdr io.Reader) (servers []models.Server, err error) {
+	var jsonServers []server
+	decoder := json.NewDecoder(rdr)
+	if err := decoder.Decode(&jsonServers); err != nil {
+		return nil, fmt.Errorf("failed unmarshaling resp body: %w", err)
+	}
+
+	servers = make([]models.Server, 0, len(jsonServers))
```

Comment:
From the current code, there can be up to 2*json servers (1 openvpn, 1 wireguard)

```suggestion
	const maxServers = 2 * len(jsonServers)
	servers = make([]models.Server, 0, maxServers)
```